sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/jsonExpressions.scala

```diff
@@ -544,34 +544,27 @@ case class JsonToStructs(
       timeZoneId = None)
```

Comment:
Pleas also update the comment of `JsonToStructs`:

`Converts an json input string to a [[StructType]] or [[ArrayType]] of [[StructType]]s`.